plugins/wazo-yealink/v86/common.py

```diff
@@ -569,6 +571,7 @@ def synchronize(self, device, raw_config):
 
     else:
         # backward compatibility with older wazo-provd server
+        # TODO: Should this be deleted? It's untestable.
```

Comment:
IMHO yes. This has not been used since very old versions of provd AFAIK.

Reply:
Ok, I think that might be a good idea then. Now or later? Is there a procedure for removing it? I assume we should bump all the versions after this?

Reply:
Yes, we should bump the versions of all the affected plugins.

Reply:
I had removed it. However, I'm not sure how to document it given we'll be pushing the new repo at the same time. A bit of s strange situation now. This will be the only py2/3 plugin in the old repo.